### ISSUE TITLE
Make drop-targeting + flipping behavior consistent

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#5966]
## Feature Description
Currently, when drop-targeting into background, the flip of the element being dragged is retained. When drop-targeting into any other elements, the flip of the target is retained (once #1111 gets fixed). This behavior has not been specifically discussed would be good to confirm the expected behavior. There are different cases to consider:

- When dragging an unflipped image into a flipped shape
- When dragging a flipped masked image into a non-flipped non-masked image
- When dragging a flipped image into a non-flipped image and vice-versa
- When dragging a flipped media into a non-flipped or flipped background.

E.g. when dragging an unflipped image into a flipped shape, it might make sense to retain the flip of the shape. However, when dragging a flipped image into a non-flipped image, does something different make sense then?

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by miina** [comment]
@o-fernandez -- created this issue to confirm the expected behavior when drop-targeting elements into each other while the dragged element and/or the target element is flipped. Added some different cases to consider into the description.

**Comment by o-fernandez** [comment]
I guess the question is, is it a more natural mental model to say that "flip" is an attribute of the image, of a shape, or both? If so, in the drag and drop flow, which takes precedence? 

I'd think that: 
- in our editor, an image or video are the same as a rectangular shape with an image in it. 
- when the user flips the shape, the image just follows the shape's flip. 
- when dragging an image/shape into another image/shape, we can do what we do with animations, preserve the flip/property of the "landing" shape. 

tl;dr - we can just keep the flip status of the "landing" shape. 

Does this make sense? @samitron7 for UX validation that this makes sense? In particular, I'm not sure if we have a good reason to have a separate flip for shapes and images within the shape, and whether the user would have ways to control those two separately.

**Comment by miina** [comment]
@barklund This was fixed within https://github.com/google/web-stories-wp/pull/7452, right?

**Comment by miina** [comment]
Closing as fixed within #7452
